apps/server/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "server",
-	"version": "0.2.1",
```

Comment:
If this is just linting errors, do we need a new version/build?

Reply:
There's no need to generate a new build for this, this change shouldn't impact functionality. Will change version back to `0.2.1`